openstack/placement/templates/stamp-alembic-job.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if and (not .Values.mariadb.enabled) .Values.api_db.initial_stamp -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "placement-stamp-alembic"
+  labels:
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/version: {{ default "unknown" .Chart.AppVersion }}
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
```

Comment:
What do these do? I don't remember seeing them being set explicitly before.

And why don't we have any of the standard labels we usually set e.g.
```
  labels:
    system: openstack
    type: api
    component: placement
```

Reply:
> What do these do? I don't remember seeing them being set explicitly before.

As I understand it, they are the [recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels). 
The labels you mention are unfortunately not uniformly set across the services, so I abandoned them in favour of the external standard. Or to me more correct: I used a file as template, where someone else has taken that decision and I just went with it.

Also, one note: Technically, it isn't part of the helm-chart installation, so it raises the question, if those labels even apply.
But also, since it is not part of the installation, I consider that question rather low priority.